### ISSUE TITLE
Would it be possible to add the names of the liverys using the native "GET_LIVERY_NAME" to make use of the names configured in the "carcols" files instead of numbers?

### ISSUE THREAD

**Issue by NiziulLuizin** [MAFINS/MenyooSP#539]
I think this would make more sense than using the numerals in the liverys, an example of this is the submenu that contains the roof livery, it's something that doesn't seem to take much time and has a great impact on the mod.

links that can be useful:

- [Carcols - Livery Names](https://gtamods.com/wiki/Carcols.ymt#liveryNames);
- [GET_LIVERY_NAME](https://nativedb.dotindustries.dev/natives/0xB4C7A93837C91A1F);

**Comment by metoxys** [comment]
Suggestion if this gets implemented: Display both, like `Name (number)` or `Number (name)`  
There are practical reasons to know the number

**Comment by NiziulLuizin** [comment]
Exactly, it would be nice if you could use the following layout: add a division with the liverys with their numerals and another division with the names of the liverys. 

A similar layout used in the part about spawning a saved vehicle:

![image](https://user-images.githubusercontent.com/74280280/226191650-6021921a-1702-41d1-a3c3-9ba17ce7cdb7.png)
